helm_deploy/hmpps-temporary-accommodation-api/values.yaml

```diff
@@ -0,0 +1,60 @@
+---
+generic-service:
+  nameOverride: hmpps-temporary-accommodation-api
+
+  replicaCount: 4
+
+  image:
+    repository: quay.io/hmpps/hmpps-temporary-accommodation-api
+    tag: app_version    # override at deployment time
+    port: 8080
+
+  ingress:
+    enabled: true
+    host: app-hostname.local    # override per environment
+    tlsSecretName: hmpps-temporary-accommodation-api-cert
+    contextColour: green
+
+  # Environment variables to load into the deployment
+  env:
+    JAVA_OPTS: "-Xmx512m"
+    SERVER_PORT: "8080"
+    SPRING_PROFILES_ACTIVE: "logstash"
+    APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
+    APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
+    SPRING_DATASOURCE_DRIVERCLASSNAME: org.postgresql.Driver
+    SPRING_JPA_DATABASE: postgresql
+    SPRING_DATASOURCE_URL: "jdbc:postgresql://${DB_HOST}/${DB_NAME}"
+
+  # Pre-existing kubernetes secrets to load as environment variables in the deployment.
+  # namespace_secrets:
+  #   [name of kubernetes secret]:
+  #     [name of environment variable as seen by app]: [key of kubernetes secret to load]
+
+  namespace_secrets:
+    hmpps-temporary-accommodation-api:
```

Comment:
We copied this file and renamed every instance of `hmpps-approved-premises-api` with `hmpps-temporary-accommodation-api`. We aren't super confident about this so would appreciate another pair of eyes. Should any of these values reflect the repository name instead?